src/components/reusable/textInput/textInput.scss

```diff
@@ -63,6 +63,10 @@ input[type='search']::-webkit-search-results-decoration {
   appearance: none;
 }
 
+input[type='search'] {
+  background-color: var(--kd-color-background-forms-secondary);
```

Comment:
Hmm why is search having a different background color compared to every other input type? I would raise this with the designer.

Reply:
@brian-patrick-3  The dropdown I was referring too have different background 
https://www.figma.com/design/CQuDZEeLiuGiALvCWjAKlu/branch/qMpff4GuFUEcsMUkvacS3U/Applications---Component-Library?node-id=4622-209539&node-type=instance&m=dev

Reply:
Chandan updated the UX and changes are in place.